src/main/java/seedu/trackermon/logic/parser/CliSyntax.java

```diff
@@ -9,5 +9,10 @@
     public static final Prefix PREFIX_NAME = new Prefix("n/");
     public static final Prefix PREFIX_STATUS = new Prefix("s/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
+    public static final Prefix PREFIX_SORT_NAME_ACS = new Prefix("sna/");
+    public static final Prefix PREFIX_SORT_STATUS_ACS = new Prefix("ssa/");
+    public static final Prefix PREFIX_SORT_NAME_DES = new Prefix("snd/");
+    public static final Prefix PREFIX_SORT_STATUS_DES = new Prefix("ssd/");
+    public static final Prefix PREFIX_SORT_STATUS_ORD = new Prefix("so/");
```

Comment:
Just curious, what is `PREFIX_SORT_STATUS_ORD`?

Reply:
For sorting status then name if both name and status are being sorted at the same time

Reply:
Shouldn't sorting name and status be just putting 2 of the prefix together?